conf/messages.cy

```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
 ukInsuranceGains.agent.none = No, my client did not make any UK insurance gains
 ukInsuranceGains.checkYourAnswersLabel = ukInsuranceGains
 ukInsuranceGains.error.required = Select if you made gains from UK insurance policies or voided ISAs, or ‘No, I did not make any UK insurance gains’
-ukInsuranceGains.agent.error.required = Select if your client made gains from UK insurance policies or voided ISAs, or ‘No, my client did not make any UK insurance gains’
\ No newline at end of file
+ukInsuranceGains.agent.error.required = Select if your client made gains from UK insurance policies or voided ISAs, or ‘No, my client did not make any UK insurance gains’
+
+ukDividendsSharesAndLoansFromLimitedCompanies.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.summary = Understanding dividends
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.p1 = You may get a dividend payment if you own shares in a company. Dividends can be cash or ‘stock’ (shares). You can earn a total of £1,000 from UK and non-UK dividends each year without paying tax. You must tell us about all of your dividend income, even if it’s less than £1,000.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.taxOnDividends.linkText = Learn more about tax on dividends (opens in a new tab)
```

Comment:
(opens in a new tab) text comes as part of the link component, could you please remove that text from the end of this message.

Reply:
Yeah I just realised this once you mentioned it, thanks. I'll remove this in my next commit.